re.sonny.Workbench.Devel.json

```diff
@@ -3,7 +3,7 @@
   "runtime": "org.gnome.Sdk",
   "runtime-version": "44",
   "sdk": "org.gnome.Sdk",
-  "sdk-extensions": ["org.freedesktop.Sdk.Extension.vala"],
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.vala","org.gnome.Sdk.Docs"],
```

Comment:
we will need this in `re.sonny.Workbench.json` as well

also, this doesn't get installed automatically so we need to copy the files into the Flatpak bundle

you can do so like this:

https://github.com/sonnyp/Workbench/pull/273/files#diff-cfaac9a05c81f5033c5692a6f05fa4c50383a5f371d10b61bb9caf74b66ec0e5R144-R147

And you can test it works correctly by using "Build" then "Export" and finally installing the `.flatpak`

Reply:
What should I be copying for the docs sdk? I can't find anything else in ``/usr/lib/sdk/`` other than ``vala``.

Reply:
you should do something like `cp -a /usr/share/docs/. /app/usr/share/docs/`

and then use the `/app/usr/share/docs` path